server/test/controllers/api/v1/cell_output_lock_scripts_controller_test.rb

```diff
@@ -4,31 +4,35 @@ module Api
   module V1
     class CellOutputLockScriptsControllerTest < ActionDispatch::IntegrationTest
       test "should get success code when call show" do
-        cell_output = create(:cell_output, :with_full_transaction)
+        block = create(:block, :with_block_hash)
```

Comment:
Why not initialize in `setup` ?

Reply:
`setup` will runs before every test. but there is one test no need to create cell output.

Reply:
I see a lot of those two lines in this file, and if you don't need in every test case, also can use lazy load.

Reply:
Thank you, I will fix this.

Reply:
@classicalliu I don't like use lazy load in test, so I solved this problem with the `create_cell_output ` method. please review again.